api_v3/lib/syndication/KalturaSyndicationFeedRenderer.php

```diff
@@ -183,7 +183,13 @@ public function __construct($feedId, $feedProcessingKey = null, $ks = null, $sta
 		}
 		else
 		{
-			$this->baseCriteria->addAnd(entryPeer::TYPE, array(entryType::MEDIA_CLIP, entryType::MIX), Criteria::IN);
+			$mediaEntrytypes = array(entryType::MEDIA_CLIP, entryType::MIX);
+			$partner = PartnerPeer::retrieveByPK($this->getSyndicationFeedDb()->getPartnerId());
+			$liveEntriesView = $partner->getEnabledService(KalturaPermissionName::FEATURE_LIVE_ENTRIES_IN_FEED);
+			if($liveEntriesView) {
```

Comment:
@TalDubovKaltura please refer to https://kaltura.atlassian.net/wiki/spaces/BEC/pages/80872310/PHP+coding+convention
We use brackets - 
<something>
{
}
rather than 
<something>{
}

Reply:
Done.